### PR TITLE
update to retail to https

### PR DESCRIPTION
**Pull request by stephengroat**
working only, not all

### CODE REVIEW

File: _data/retail.yml [hunk 1]
```diff
@@ -81,13 +81,13 @@ websites:
       facebook: Costco
 
     - name: Dell
-      url: http://www.dell.com
+      url: https://www.dell.com
```
Comment (jamcat22) [inline _data/retail.yml]: This redirects down to HTTP. Should we have the HTTPS site listed and hope they eventually support it fully?

File: _data/retail.yml [hunk 2]
```diff
@@ -99,7 +99,7 @@ websites:
       twitter: EBGamesAus
 
     - name: eBay
-      url: http://ebay.com
+      url: https://ebay.com
```
Comment (jamcat22) [inline _data/retail.yml]: Redirects to `https://www.ebay.com`

File: _data/retail.yml [hunk 3]
```diff
@@ -36,8 +36,8 @@ websites:
       sms: Yes
       software: Yes
       exceptions:
-          text: "See http://support.apple.com/kb/HT5593 for a list of supported SMS carriers."
-      doc: http://support.apple.com/kb/ht5570
+          text: "See https://support.apple.com/kb/HT5593 for a list of supported SMS carriers."
```
Comment (jamcat22) [inline _data/retail.yml]: Redirects to `https://support.apple.com/kb/HT202656`

File: _data/retail.yml [hunk 4]
```diff
@@ -36,8 +36,8 @@ websites:
...
+      doc: https://support.apple.com/kb/ht5570
```
Comment (jamcat22) [inline _data/retail.yml]: Redirects to `https://support.apple.com/kb/HT204152`

**Review by jamcat22 — CHANGES_REQUESTED** [review]
Other than what's listed below, looks good to me!

**Comment by Carlgo11** [comment]
Could we use this PR to change to HTTPS on all categories? Maybe even clean up old sites?

File: _data/retail.yml (hunk 1 above)
Comment (mxxcon) [reply _data/retail.yml]: I'd say yes.

Comment (stephengroat) [reply _data/retail.yml]: currently does not redirect for me

File: _data/retail.yml (hunk 3 above)
Comment (stephengroat) [reply _data/retail.yml]: updated

File: _data/retail.yml (hunk 4 above)
Comment (stephengroat) [reply _data/retail.yml]: updated

**Comment by conorgil** [comment]
@stephengroat I think https://github.com/2factorauth/twofactorauth/pull/3103 being merged means this issue can be closed. Thoughts?

**Comment by jamcat22** [comment]
@conorgil the Apple doc and exception links and the Dell link haven't been changed by #3103.

**Comment by stephengroat** [comment]
@jamcat22 the apple links were updated by 95c0070cdc3dba41333958d4f73a8155c53c13ed

i don't really know what to do about the dell link, but that's updated in 648283850a4b43fe2f88bbd1f529cba3fb170e66 as well

**Comment by jamcat22** [comment]
@stephengroat I was saying those are the two changes that are in this PR that weren't included in #3103. Should we make a new PR for those, or rebase and merge this one?

**Comment by stephengroat** [comment]
up to you, you've got edit ability on mine

**Pull request closed without merging**